src/dev/build/lib/config.test.ts

```diff
@@ -45,10 +45,23 @@ const versionInfo = jest.requireMock('./version_info').getVersionInfo();
 
 expect.addSnapshotSerializer(createAbsolutePathSerializer());
 
-const setup = async ({ targetAllPlatforms = true }: { targetAllPlatforms?: boolean } = {}) => {
+const setup = async ({
+  targetAllPlatforms = true,
+  darwinX64 = false,
```

Comment:
Do we think this list will grow large? If so I can perhaps see a benefit of creating a param of "platform" of type string and we define a constants file somewhere.

Reply:
Will wrap it in an interface.